pybind11_stubgen/__init__.py

```diff
@@ -334,27 +341,31 @@ def to_output_and_subdir(
 
     if root_suffix is None:
         return out_dir.joinpath(*module_path[:-1]), None
+
```

Comment:
This diff looks weird, its just a removal of the "else" and indentation, identical functionality